src/SOS/Strike/sos_stacktrace.h

```diff
@@ -166,6 +166,16 @@ HRESULT CALLBACK _EFN_GetManagedObjectFieldInfo(
     PULONG pOffset
     );
 
+// _EFN_GetManagedThread - return the managed thread object for the os thread id
+//
+// osThreadId - os thread id
+// pManagedObject - managed thread object returned
```

Comment:
For naming "Managed thread object" generally means an instance of System.Threading.Thread, whereas I think you are referring to the Thread* pointer? Perhaps calling it the 'runtime thread' or 'managed thread' would be better? "object" is the part of the name that makes me assume you are talking about the System.Threading.Thread.

Reply:
I removed/renamed the "object" from the code.